src/js/reducers/globalReducer.js

```diff
@@ -1,4 +1,5 @@
 import constants from '../services/constants';
+import Cookies from 'js-cookie';
```

Comment:
Why we need that lib, when we already has this func in common?

Reply:
Updated, using common instead.